app/models/event.rb

```diff
@@ -42,4 +42,11 @@ def format_time_range
     end
   end
 
+  def total_registerted
+    if registrations.present?
+      registrations.map(&:guests_registered).reduce(:+) + users.size
```

Comment:
Why not just use + registrations.size?

Reply:
I think this is maybe a little more clear, but maybe not?

Using `registrations` might save use a sql call every now and again though, so I'm fine changing it.